apps/students/urls.py

```diff
@@ -8,9 +8,13 @@
     path('new/', views.new_student, name='new_student'),
     path('profile/<int:pk>/', views.profile, name='profile'),
     path('attendance/', views.attendance, name='attendance'),
-    path('ajax/attendance/', views.ajax_attendance, name='ajax_attendance'),
     # path('attendance/view/', views.view_attendance, name='view_attendance'),
     path('profile/update/<int:pk>/', views.update_profile, name='update_profile'),  # Not needed, update in profile/ only
 
     path('update_from_sheets/', views.update_from_sheets, name='update_from_sheets'),
+
+    # AJAX calls
+    path('ajax/ajax_attendance_fetch_students/', views.ajax_attendance_fetch_students, name='ajax_attendance_fetch_students'),
```

Comment:
Don't repeat `ajax` in the endpoint. `ajax/fetch_students/` and `ajax/mark_attendance` should work fine. Replace `/` with `_` for view name and URL namespace.